### PR TITLE
[IMP] theme_kiddo, theme_vehicule: add images_peview_theme in manifest

### PR DESCRIPTION
**Pull request by JKE-be**
Example of how to communicate image to be used by IAP in preview when use_theme_image is checked on iap.
//cc @Cocographique

### CODE REVIEW

**Comment by JKE-be** [comment]
@robodoo r+

//cc @Cocographique

File: theme_kiddo/__manifest__.py [hunk 1]
```diff
@@ -17,11 +17,47 @@
         'views/snippets/s_three_columns.xml',
         'views/snippets/s_call_to_action.xml',
 
-        ],
+    ],
     'images': [
         'static/description/Kiddo_description.png',
         'static/description/kiddo_screenshot.jpg',
     ],
+    'images_peview_theme': {
```
Comment (qsm-odoo) [inline theme_kiddo/__manifest__.py:25]: "peview" ? :wink:

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

**Comment by JKE-be** [comment]
@qsm-odoo thank you ;) 
don't hesitate to r- when you are sure that it is wrong ;) 
I don't bite and I'm able to see that I'm wrong when it is the case (most of the time :p) ;)

File: theme_vehicle/__manifest__.py [hunk 2]
```diff
@@ -16,6 +16,34 @@
         'static/description/vehicle_description.png',
         'static/description/vehicle_screenshot.jpeg',
     ],
+    'images_peview_theme': {
```
Comment (rdeodoo) [inline theme_vehicle/__manifest__.py:19]: :eye:

Comment (rdeodoo) [reply theme_vehicle/__manifest__.py]: fixed this one
 I guess it can be  r+ now?

**Comment by JKE-be** [comment]
@robodoo r+

**Pull request closed without merging**